templates/Makefile.build-harness-bootstrap

```diff
@@ -11,7 +11,7 @@ export BUILD_HARNESS_PATH ?= $(shell until [ -d "$(BUILD_HARNESS_PROJECT)" ] ||
 -include $(BUILD_HARNESS_PATH)/Makefile
 export BUILD_HARNESS_EXTENSIONS_HOST ?= github.com
 export BUILD_HARNESS_EXTENSIONS_HOST_AUTH ?= false
-export BUILD_HARNESS_EXTENSIONS_ORG ?= multicloud-ops
+export BUILD_HARNESS_EXTENSIONS_ORG ?= sethpeterson
```

Comment:
We probably don't want to merge your test change. 
if you want to test these kind of changes locally you can just do 
```export BUILD_HARNESS_EXTENSIONS_ORG=sethpeterson
make <whatever>
```

Reply:
Thanks!